cairo/tests/ethereum/cancun/test_state.py

```diff
@@ -68,6 +79,87 @@ def state_and_address_and_optional_key(
     return state, address, key
 
 
+@composite
+def state_with_snapshots(draw):
```

Comment:
Could use those composites function for all state and transient storage generation ?

Reply:
There's only in these specific scenarios that I need a properly laid-out state... or for a full run. So i'm not so sure yet, because these strategies are long to generate 🤔